docs/reference-guides/rancher-manager-architecture/rancher-server-and-components.md

```diff
@@ -20,6 +20,14 @@ A high-availability Kubernetes installation is recommended for production.
 
 A Docker installation of Rancher is recommended only for development and testing purposes. The ability to migrate Rancher to a high-availability cluster depends on the Rancher version.
 
+The resources that Rancher deploys when we go as a single node Rancher deployment 
+
+![Rancher Components ](https://user-images.githubusercontent.com/37767537/233318973-880855fa-5271-468e-95eb-e1290d5c91c3.png)
+
+You can check this using the kubectl command in the namespaces where you have deployed Rancher. You can achieve the same using the CLI or going inside Rancher admin account and filtering the namespaces.
```

Comment:
Could you include the exact command and step-by-step instructions for filtering the namespaces?